### DISCUSSION TITLE
Themes 101

### DISCUSSION THREAD

**Discussion by sonicsmooth** [AvaloniaUI/Avalonia#10053] [Q&A] [answered]
I absolutely cannot figure out how to change themes.  I can change styles of individual controls, etc., but I'm unable to change from the default Fluent theme to anything else.  Specifically, I want to enable either Default or Simple theme.  I really don't like Fluent.

I added `<PackageReference Include="Avalonia.Themes.Simple" Version="0.10.18" />` to project and `<SimpleTheme Mode="Light"/>` to App.axaml and I get this:
![image](https://user-images.githubusercontent.com/672429/213964944-9fd75618-ad89-4c2c-a53f-192c06761c24.png)

In this case, dependencies looks like this: 
![image](https://user-images.githubusercontent.com/672429/213965318-03575a76-a57e-41b4-944a-271c88b6e115.png)

When instead I use version 11.0.0,  ` <PackageReference Include="Avalonia.Themes.Simple" Version="11.0.0-preview3" />`  (everything else at 0.10.18) I get this 
![image](https://user-images.githubusercontent.com/672429/213964686-cc67cabb-e6fe-4f6c-aff9-89edc9885329.png)

In this case, dependencies look like this:
![image](https://user-images.githubusercontent.com/672429/213965478-4ea1b12c-774a-4933-8399-5140814d7dd1.png)

Commenting out the `<FluentTheme Mode="Light"/>` from App.axaml leads to other errors.

It looks like I have something installed, but the words "not available in this source" are not encouraging.
![image](https://user-images.githubusercontent.com/672429/213963234-fcb52bb7-a34e-4ff2-be7d-9d958e514dff.png)

There's also these two but I don't think they're official Avalonia, and anyway my ignorance continues with this package.  How would I install/use it? 
![image](https://user-images.githubusercontent.com/672429/213963647-e2eea8fd-9fe0-4480-9ad5-2f98b9b12c55.png)

I also see the source in github [Avalonia](https://github.com/AvaloniaUI/Avalonia)/[src](https://github.com/AvaloniaUI/Avalonia/tree/master/src)/[Avalonia.Themes.Simple](https://github.com/AvaloniaUI/Avalonia/tree/master/src/Avalonia.Themes.Simple)/SimpleTheme.xaml but I don't know what to do with that.

Finally, there is [this](https://github.com/AvaloniaUI/Avalonia/discussions/9314) recent post that is the source of much in this message, but there is a lot of context I need to really understand.

I'm not even trying to change themes at runtime.  Just set and forget.

Do I need to upgrade everything to 11.0?  How do I do this?

I'm thinking this really should just be a one-liner.  But there is no clear canonical "how to" on themes.

What special theory or high level concept am I missing here?

How do I set Simple or Default theme?

thanks

**Comment by sonicsmooth** [comment]
I upgrade everything to 11.0, and that seemed to do the trick.  I can invoke simple theme now with basic controls.

**Comment by maxkatz6** [comment, marked as answer]
For the stable 0.10.18 version you need to use StyleInclude to get simple/default theme:
https://github.com/AvaloniaUI/Avalonia/blob/0.10.18/samples/BindingDemo/App.xaml#L6
And you didn't need to include PackageReference for the themes.

In 11.0 it was renamed to the Simple theme (as we have two defaults and it was confusing), and now there is a simplier syntax to include it, using `<SimpleTheme />`.
And PackageReference for the specific theme is required now.
